### ISSUE TITLE
FutureWarning in np.linalg.lstsq

### ISSUE THREAD

**Issue by rmarkello** [ME-ICA/tedana#36]
There are a lot of calls to `numpy.linalg.lstsq` throughout all the codebase. Currently, each time it is called it throws a `FutureWarning`:

```python-traceback
FutureWarning: `rcond` parameter will change to the default of machine precision times ``max(M, N)`` where M and N are the input matrix dimensions.
  To use the future default and silence this warning we advise to pass `rcond=None`, to keep using the old, explicitly pass `rcond=-1`.
```

It might be worth examining whether there are meaningful differences between `rcond=None` and `rcond=-1` for the code. Whichever is decided, it would be good to through and add `rcond` as a keyword argument to every call to inhibit the warnings.

**Comment by emdupre** [comment]
Yes, please, let's fix these. I'm sort of numb to the sea of warnings at this point, but I imagine it's very off-putting for new users !

**Comment by rmarkello** [comment]
Closed with #41!
